translations/stripes-acq-components/en.json

```
@@ -121,6 +122,20 @@
   "referenceNumbers.refNumberType.refNumber": "Vendor order reference number",
   "referenceNumbers.refNumberType.titleNumber": "Vendor title number",
 
+  "shortcut.createRecord": "Create a new record",
```

```suggestion
  "shortcut.new": "Create a new record",
```
and other `name`s too